.github/actions/setup-project/action.yml

```
@@ -19,12 +19,13 @@ runs:
       uses: astral-sh/setup-uv@v6
       with:
         python-version: ${{ inputs.python-version }}
+        cache-dependency-glob: "**/*requirements*.txt,**/*requirements*.in,**/*constraints*.txt,**/*constraints*.in,**/pyproject.toml,**/uv.lock"
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)